src/games/strategy/engine/framework/mapDownload/MapDownloadProperties.java

```diff
@@ -0,0 +1,53 @@
+package games.strategy.engine.framework.mapDownload;
```

Comment:
This class reads the local config file that tells the game engine where to go get the "triplea_maps.xml" file.